### PR TITLE
Update ERC20 bridge restricted with last specs

### PR DESCRIPTION
**Pull request by jeremyletang**
Update implementation to:

### Update limits
- remove deposit_minimum
- remove deposit_maximum
- merge function to set limits (1 call for both WithdrawThreshold and MaximumLifetimeDeposit)
- send event on Set limits
- send event on SetWithdrawDelay

### Fix deposit exemption:
- remove exemption_lister support
- only sender can exempt themselves

### CODE REVIEW

File: contracts/ERC20_Bridge_Logic_Restricted.sol [hunk 1]
```diff
@@ -112,28 +79,43 @@ contract ERC20_Bridge_Logic_Restricted is IERC20_Bridge_Logic_Restricted {
...
-      bytes memory message = abi.encode(asset_source, lifetime_limit, nonce, 'set_lifetime_deposit_max');
+      bytes memory message = abi.encode(asset_source, lifetime_limit, threshold, nonce, 'set_asset_limits');
       require(IMultisigControl(multisig_control_address()).verify_signatures(signatures, message, nonce), "bad signatures");
-      asset_deposit_lifetime_limit[asset_source] = lifetime_limit;
+      if (asset_deposit_lifetime_limit[asset_source] != lifetime_limit) {
+	  asset_deposit_lifetime_limit[asset_source] = lifetime_limit;
+      }
+      if (withdraw_thresholds[asset_source] != threshold) {
+	  withdraw_thresholds[asset_source] = threshold;
+      }
```
Comment (jeremyletang) [inline contracts/ERC20_Bridge_Logic_Restricted.sol]: I'm not sure the ifs are necessary here, added them so we don;'t write on the map for no reason, but that may be overkill

Comment (C0deMunk33) [reply contracts/ERC20_Bridge_Logic_Restricted.sol]: this is overkill

File: contracts/ERC20_Bridge_Logic_Restricted.sol [hunk 2]
```diff
@@ -112,28 +79,43 @@ contract ERC20_Bridge_Logic_Restricted is IERC20_Bridge_Logic_Restricted {
 
     // depositor => is exempt from deposit limits
     mapping(address => bool) exempt_depositors;
```
Comment (C0deMunk33) [inline contracts/ERC20_Bridge_Logic_Restricted.sol:81]: This mapping is unused if we remove the deposit limits

**Review by C0deMunk33 — APPROVED** [review]
I've gone through the changes and I believe we are good.

**Pull request merged**